src/test/java/core/basesyntax/strategy/impl/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.dao.FruitsDao;
+import core.basesyntax.dao.FruitsDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.FruitHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static FruitTransaction fruitTransaction;
+    private static OperationStrategy operationStrategy;
+    private static FruitsDao fruitsDao;
```

Comment:
In handlers tests you can access storage directly, because there is a possibility that dao was implemented wrong and we can get error which we do not want to get.